usage-collection/src/main/java/com/sequenceiq/cloudbreak/usage/MeteringEventProcessor.java

```diff
@@ -0,0 +1,74 @@
+package com.sequenceiq.cloudbreak.usage;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.cloudera.thunderhead.service.metering.events.MeteringEventsProto;
+import com.sequenceiq.cloudbreak.auth.crn.Crn;
+import com.sequenceiq.cloudbreak.auth.crn.RegionAwareInternalCrnGeneratorFactory;
+import com.sequenceiq.cloudbreak.sigmadbus.model.DatabusRequest;
+import com.sequenceiq.cloudbreak.sigmadbus.model.DatabusRequestContext;
+import com.sequenceiq.cloudbreak.usage.metering.MeteringDatabusRecordProcessor;
+
+@Service
+public class MeteringEventProcessor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MeteringEventProcessor.class);
+
+    private static final int METERING_EVENT_VERSION = 3;
+
+    private final MeteringDatabusRecordProcessor meteringDatabusRecordProcessor;
+
+    private final RegionAwareInternalCrnGeneratorFactory internalCrnGeneratorFactory;
+
+    public MeteringEventProcessor(MeteringDatabusRecordProcessor meteringDatabusRecordProcessor,
+            RegionAwareInternalCrnGeneratorFactory internalCrnGeneratorFactory) {
+        this.meteringDatabusRecordProcessor = meteringDatabusRecordProcessor;
+        this.internalCrnGeneratorFactory = internalCrnGeneratorFactory;
+    }
+
+    public void processEvent(MeteringEventsProto.MeteringEvent event) {
```

Comment:
Please reduce visibility to package-private and add `@VisibleForTesting`, since it does not seem to be called anywhere else.